spec/paperclip/storage/fog_spec.rb

```diff
@@ -418,6 +418,9 @@ def custom_method
           assert @connection.directories.get(@dynamic_fog_directory).inspect
         end
 
+        it "provides an url using dynamic bucket name" do
+          assert_match(/^https:\/\/dynamicpaperclip.s3.amazonaws.com\/avatars\/5k.png\?\d*$/, @dummy.avatar.url)
```

Comment:
Sorry my last comment was a bit sarcastic. I'm leaving this as is for now as it matches the established code style.